ipfailover/keepalived/lib/failover-functions.sh

```diff
@@ -34,6 +34,23 @@ function cleanup() {
   exit 0
 }
 
+function unconfigure_failover() {
+  echo "  - Removing ip_vs module ..."
+  modprobe -r ip_vs
+
+  echo "  - Removing keepalived multicast iptables rule ..."
+  chain="${HA_IPTABLES_CHAIN:-""}"
+  if [[ -n ${chain} ]]; then
+    if iptables -S | grep 224.0.0.18 > /dev/null 2>&1 ; then
+      echo "  - Removing keepalived multicast iptables rule ..."
+      iptables -D ${chain} 1 -d 224.0.0.18/32 -j ACCEPT
+    fi
+  fi
+
+  cleanup $(pidof /usr/sbin/keepalived)
```

Comment:
Is pidof available?  You might need to change the keepalived-ipfailover Dockerfile to install the procps-ng package.

Reply:
added